tests/crds_test.go

```diff
@@ -157,3 +158,14 @@ func appendFile(t *testing.T, filename, content string) {
 
 	_, err = f.WriteString(content)
 }
+
+func runRequireNoError(t *testing.T, cmd *exec.Cmd) {
```

Comment:
nit: you could also use https://pkg.go.dev/os/exec#Cmd.CombinedOutput to do this.

Reply:
Ah thanks, I was looking for that, but didn't see it!